### PR TITLE
Wait for DB Docker image ports to be ready before running Cypress tests

### PR DESCRIPTION
**Pull request by camsaul**
That would hopefully fix most of the "connection refused" errors we see on occasion if the tests start running before the image is ready

### CODE REVIEW

**Review by robdaemon — APPROVED** [review]

**Comment by robdaemon** [comment]
LGTM, @nemanjaglumac wanna take a look?

File: .circleci/config.yml [hunk 1]
```diff
@@ -1336,6 +1340,9 @@ workflows:
           cypress-group: "mongo"
           only-single-database: true
           test-files-location: frontend/test/metabase-db/mongo
+          before-steps:
+            - wait-for-port:
+                port: 27019
```
Comment (nemanjaglumac) [inline .circleci/config.yml:1345]:
```suggestion
                port: 27017
```

Comment (camsaul) [reply .circleci/config.yml:1345]: oops

Comment (robdaemon) [reply .circleci/config.yml]: oops I missed that too

**Pull request merged**